ui/app/components/auth-form.js

```diff
@@ -3,35 +3,28 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Ember from 'ember';
 import { next } from '@ember/runloop';
 import { service } from '@ember/service';
 import { match, or } from '@ember/object/computed';
 import { dasherize } from '@ember/string';
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { allSupportedAuthBackends, supportedAuthBackends } from 'vault/helpers/supported-auth-backends';
-import { task, timeout } from 'ember-concurrency';
+import { task } from 'ember-concurrency';
 import { waitFor } from '@ember/test-waiters';
 import { v4 as uuidv4 } from 'uuid';
 
 /**
  * @module AuthForm
- * The `AuthForm` is used to sign users into Vault.
+ * The AuthForm is used to sign users into Vault. All properties are passed in via query params.
  *
- * @example ```js
- * // All properties are passed in via query params.
- * <AuthForm @wrappedToken={{wrappedToken}} @cluster={{model}} @namespace={{namespaceQueryParam}} @selectedAuth={{authMethod}} @onSuccess={{action this.onSuccess}}/>```
+ * @example
+ * <AuthForm @wrappedToken={{wrappedToken}} @cluster={{model}} @namespace="admin" @selectedAuth="token" />
  *
  * @param {string} wrappedToken - The auth method that is currently selected in the dropdown.
```

Comment:
nit: that's not what a wrapped token is 😂

Reply:
Hahaha nope, it sure isn't. I was planning to audit these args while here towards the end of the PR!